processing file asmFiles/test.loadstore.asm (1 of 2)
processing file asmFiles/test.rtype.asm (2 of 2)
processing file asmFiles/test.loadstore.asm (1 of 2)
processing file asmFiles/test.rtype.asm (2 of 2)
processing file asmFiles/test.loadstore.asm (1 of 2)
--- memsim.hex	2015-02-06 03:37:59.131025300 -0500
+++ memcpu.hex	2015-02-06 03:36:16.301156739 -0500
@@ -1,20 +1,39 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
-:0400030034E7BEEF31
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC47000CF3
-:04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
+:040000003401D2698C
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:040020000000F7F9EC
+:040021000000126168
+:0400220000000009D1
+:0400230000010A5A74
+:0400240000007F3920
+:04002500FFFFDA708F
+:04002600000037F8A7
+:04002700000021565E
+:04002800000D269011
+:04002900000006933A
+:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.rtype.asm (2 of 2)
processing file asmFiles/test.loadstore.asm (1 of 2)
processing file asmFiles/test.loadstore.asm (1 of 2)
--- memsim.hex	2015-02-06 03:39:09.285982274 -0500
+++ memcpu.hex	2015-02-06 03:39:09.269981828 -0500
@@ -1,4 +1,4 @@
-:04000000340100F0D7
+:04000000DEAD000071
 :040001003402008045
 :040002003C07DEAD2C
 :0400030034E7BEEF31
@@ -10,10 +10,6 @@
 :04000900AC450008FA
 :04000A00AC47000CF3
 :04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
 :04003C000000733716
 :04003D000000270197
 :04003E000000133774
processing file asmFiles/test.rtype.asm (2 of 2)
--- memsim.hex	2015-02-06 03:39:10.915027716 -0500
+++ memcpu.hex	2015-02-06 03:39:10.898027242 -0500
@@ -1,4 +1,3 @@
-:040000003401D2698C
 :04000100340237F19D
 :040002003415008031
 :04000300341600F0BF
@@ -25,15 +24,4 @@
 :04001800AEAB00206B
 :04001900AEAC002465
 :04001A00FFFFFFFFE6
-:040020000000F7F9EC
-:040021000000126168
-:0400220000000009D1
-:0400230000010A5A74
-:0400240000007F3920
-:04002500FFFFDA708F
-:04002600000037F8A7
-:04002700000021565E
-:04002800000D269011
-:04002900000006933A
-:04003C00FFFF0806B4
 :00000001FF
processing file asmFiles/test.loadstore.asm (1 of 2)
--- memsim.hex	2015-02-06 03:39:58.888365895 -0500
+++ memcpu.hex	2015-02-06 03:39:58.821364026 -0500
@@ -1,4 +1,4 @@
-:04000000340100F0D7
+:04000000DEAD000071
 :040001003402008045
 :040002003C07DEAD2C
 :0400030034E7BEEF31
@@ -10,11 +10,7 @@
 :04000900AC450008FA
 :04000A00AC47000CF3
 :04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
+:04003C00340100F09B
 :04003D000000270197
 :04003E000000133774
 :00000001FF
processing file asmFiles/test.rtype.asm (2 of 2)
--- memsim.hex	2015-02-06 03:40:00.518411363 -0500
+++ memcpu.hex	2015-02-06 03:40:00.502410917 -0500
@@ -1,4 +1,3 @@
-:040000003401D2698C
 :04000100340237F19D
 :040002003415008031
 :04000300341600F0BF
@@ -25,15 +24,5 @@
 :04001800AEAB00206B
 :04001900AEAC002465
 :04001A00FFFFFFFFE6
-:040020000000F7F9EC
-:040021000000126168
-:0400220000000009D1
-:0400230000010A5A74
-:0400240000007F3920
-:04002500FFFFDA708F
-:04002600000037F8A7
-:04002700000021565E
-:04002800000D269011
-:04002900000006933A
-:04003C00FFFF0806B4
+:043FFF00FFFFFFFFC2
 :00000001FF
processing file asmFiles/test.loadstore.asm (1 of 2)
--- memsim.hex	2015-02-06 03:40:49.945790215 -0500
+++ memcpu.hex	2015-02-06 03:40:00.502410917 -0500
@@ -1,20 +1,28 @@
-:04000000340100F0D7
-:040001003402008045
-:040002003C07DEAD2C
-:0400030034E7BEEF31
-:040004008C23000049
-:040005008C24000443
-:040006008C2500083D
-:04000700AC43000006
-:04000800AC44000400
-:04000900AC450008FA
-:04000A00AC47000CF3
-:04000B00FFFFFFFFF5
-:040020000000733732
-:0400210000002701B3
-:040022000000133790
-:04002300DEADBEEFA1
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
+:04000100340237F19D
+:040002003415008031
+:04000300341600F0BF
+:040004000022182599
+:040005000022202491
+:040006003025000F92
+:040007000022302182
+:0400080024678740A2
+:04000900008240230E
+:04000A0000A24826E2
+:04000B00382AF33F5D
+:04000C000020590077
+:04000D00002061422C
+:04000E00002268273D
+:04000F00AECD000072
+:04001000AEA300009B
+:04001100AEA4000495
+:04001200AEA500088F
+:04001300AEA6000C89
+:04001400AEA7001083
+:04001500AEA800147D
+:04001600AEA9001877
+:04001700AEAA001C71
+:04001800AEAB00206B
+:04001900AEAC002465
+:04001A00FFFFFFFFE6
+:043FFF00FFFFFFFFC2
 :00000001FF
processing file asmFiles/test.rtype.asm (2 of 2)
--- memsim.hex	2015-02-06 03:40:50.149795909 -0500
+++ memcpu.hex	2015-02-06 03:40:00.502410917 -0500
@@ -1,4 +1,3 @@
-:040000003401D2698C
 :04000100340237F19D
 :040002003415008031
 :04000300341600F0BF
@@ -25,15 +24,5 @@
 :04001800AEAB00206B
 :04001900AEAC002465
 :04001A00FFFFFFFFE6
-:040020000000F7F9EC
-:040021000000126168
-:0400220000000009D1
-:0400230000010A5A74
-:0400240000007F3920
-:04002500FFFFDA708F
-:04002600000037F8A7
-:04002700000021565E
-:04002800000D269011
-:04002900000006933A
-:04003C00FFFF0806B4
+:043FFF00FFFFFFFFC2
 :00000001FF
processing file asmFiles/test.loadstore.asm (1 of 2)
processing file asmFiles/test.loadstore.asm (1 of 2)
processing file asmFiles/test.rtype.asm (2 of 2)
processing file asmFiles/test.loadstore.asm (1 of 2)
processing file asmFiles/test.rtype.asm (2 of 2)
processing file asmFiles/test.loadstore.asm (1 of 2)
processing file asmFiles/test.rtype.asm (2 of 2)
